spec/index.html

```diff
@@ -67,7 +67,7 @@
       are comprised of a default graph and zero or more named graphs.</li>
   </ul>
 
-  <p>RDF 1.2 introduces <a>quoted triples</a> as another kind of <a>RDF term</a>
+  <p>RDF 1.2 introduces <a>triple terms</a> as another kind of <a>RDF term</a>
     which can be used as the <a>subject</a> or <a>object</a> of another <a>triple</a>.
```

Comment:
As [pointed out](https://github.com/w3c/rdf-concepts/pull/78/files#r1492917262) by @pfps, the new definition of 'RDF triple' in this PR does not permit triple terms in the subject position. Hence, this sentence here needs to be adapted.
```suggestion
    which can be used as the <a>object</a> of another <a>triple</a>.
```